catalog/pipeline/fbc-release/0.2/fbc-release.yaml

```diff
@@ -36,8 +36,16 @@ spec:
     - name: addArches
       type: string
       description: List arches to be added to be built
+    - name: requestUpdateTimeout
+      type: string
+      description: Max seconds to wait until the status is updated
   workspaces:
     - name: release-workspace
+  results:
+    - name: requestMessage
```

Comment:
how does this get set?

Reply:
The step `watch-internal-request-status` watches the InternalRequest until it completes (on fail or success) - then setting the task result. This task result is read from the pipeline when it finishes.

Reply:
Thanks to Leandro's explanation, I now understand.